.travis.yml

```diff
@@ -1,11 +1,26 @@
+sudo: false
 language: node_js
-branches:
-  only:
-    - master
-os:
-    - linux
-    - osx
 node_js:
-   - "8"
-   - "6"
-   - "4"
+  - '9'
+  - '8'
+  - '7'
+  - '6'
+  - '5'
+  - '4'
+  - '0.12'
+  - '0.10'
```

Comment:
I want to start testing from node 4, I mean LTS is 8 and < 4 should not be supported anymore.